django_project/api/urls.py

```diff
@@ -0,0 +1,58 @@
+from django.conf.urls import url
+from api.viewsets.provider import ProvidersViewSet
+from api.viewsets.qualification import QualificationViewSet
+from api.viewsets.campus import CampusViewSet
+from api.viewsets.campus_event import CampusEventViewSet
+from api.viewsets.qualification_event import QualificationEventViewSet
+from api.viewsets.saqa_qualification import SAQAQualificationViewSet
+
+
+urlpatterns = [
+    url(r'^(?P<version>(v1))/providers/$',
+        ProvidersViewSet.as_view({'get': 'list'}),
+        name='show-providers-api'),
+
+    url(r'^(?P<version>(v1))/providers/(?P<pk>\d+)/$',
+        ProvidersViewSet.as_view({'get': 'retrieve'}),
+        name='show-provider-api'),
+
+    url(r'^(?P<version>(v1))/campus/$',
+        CampusViewSet.as_view({'get': 'list'}),
+        name='show-campuses-api'),
+
+    url(r'^(?P<version>(v1))/campus/(?P<pk>\d+)/$',
+        CampusViewSet.as_view({'get': 'retrieve'}),
+        name='show-campus-api'),
+
+    url(r'^(?P<version>(v1))/qualifications/$',
+        QualificationViewSet.as_view({'get': 'list'}),
+        name='show-qualifications-api'),
+
+    url(r'^(?P<version>(v1))/qualifications/(?P<pk>\d+)/$',
+        QualificationViewSet.as_view({'get': 'retrieve'}),
+        name='show-qualification-api'),
+
+    url(r'^(?P<version>(v1))/qualification-events/$',
+        QualificationEventViewSet.as_view({'get': 'list'}),
+        name='show-qualification-events-api'),
+
+    url(r'^(?P<version>(v1))/qualification-events/(?P<pk>\d+)/$',
+        QualificationEventViewSet.as_view({'get': 'retrieve'}),
+        name='show-qualification-event-api'),
+
+    url(r'^(?P<version>(v1))/saqa-qualifications/$',
+        SAQAQualificationViewSet.as_view({'get': 'list'}),
+        name='show-saqa-qualifications-api'),
+
+    url(r'^(?P<version>(v1))/saqa-qualifications/(?P<pk>\d+)/$',
+        SAQAQualificationViewSet.as_view({'get': 'retrieve'}),
+        name='show-saqa-qualification-api'),
+
+    url(r'^(?P<version>(v1))/campus-events/$',
+        CampusEventViewSet.as_view({'get': 'list'}),
+        name='show-campus-events-api'),
```

Comment:
`show-campus-events-api` and `show-campus_event-api` are really similar name. why?